packages/1155-contracts/src/delegation/ZoraCreator1155PremintExecutorImpl.sol

```diff
@@ -190,6 +191,24 @@ contract ZoraCreator1155PremintExecutorImpl is
         );
     }
 
+    /// @notice Returns the version of the premint signature that the contract supports
+    /// @param contractAddress The address of the contract to check
+    /// @return The version of the premint signature that the contract supports.  If it doesn't support premint
+    /// returns 0
+    function supportedPremintSignatureVersion(address contractAddress) external view returns (string memory) {
+        IZoraCreator1155 creatorContract = IZoraCreator1155(contractAddress);
```

Comment:
should version be a string or an integer?

Reply:
hrrrm great q. when you sign the message, the version according to eip-712 is a string.  so maybe it makes sense for this to actually return an array saying `supportedSignatureVersions` with an array of the versions supported?

or we store internally as integers and convert to string in the eip-712 domain, and this returns a an integer which acts as a max version.

Reply:
String is fine for alignment for EIP-712. Another option would be to stringify an integer to come up.